index.html

```diff
@@ -342,5 +342,33 @@ <h5>Skills</h5>
         <hr />
       </ul>
     </aside>
+<section class="contact">
+  <form
+    action="https://formspree.io/f/xzbqggzq"
+    method="POST">
+    <div class="form-container">
+      <div class="form-header">
+        <h4>Contact me </h4>
+      </div>
+      <div class="form-desc">
+        <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
+      </div>
+      <div class="form-input" >
+      <div class="form-val">
+        <input type="text" placeholder="Name" minlength="5" maxlength="30" required/>
```

Comment:
- [ ] Although I filled all the required inputs, it returned an error 👇🏽 

![image](https://user-images.githubusercontent.com/58642949/222443163-dd982bd9-94d2-47fa-bdaf-6b15eb668bd0.png)

Please 🙏🏽 fix it for a better user experience.

Reply:
Done :heavy_check_mark: